mrmustard/lab_dev/transformations/base.py

```diff
@@ -36,6 +40,25 @@ class Transformation(CircuitComponent):
     Base class for all transformations.
     """
 
+    def inverse(self) -> Transformation:
+        r"""Returns the inverse of the transformation."""
+        if not isinstance(self.representation, Bargmann):
+            raise NotImplementedError("Only Bargmann representation is supported.")
```

Comment:
Only single mode Bargmann is supported here, am I correct?

Reply:
no, any number of modes